core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -98,12 +98,15 @@ trait RawlsApiService //(val workspaceServiceConstructor: UserInfo => WorkspaceS
 
   val baseApiRoutes = workspaceRoutes ~ entityRoutes ~ methodConfigRoutes ~ submissionRoutes ~ adminRoutes ~ userRoutes ~ billingRoutesV2 ~ billingRoutes ~ notificationsRoutes ~ servicePerimeterRoutes ~ snapshotRoutes
 
+  val instrumentedRoutes = instrumentRequest(baseApiRoutes)
+
   def apiRoutes =
     options(complete(OK)) ~
     withExecutionContext(ExecutionContext.global) { //Serve real work off the global EC to free up the dispatcher to run more routes, including status
-      baseApiRoutes
+      instrumentedRoutes
```

Comment:
`instrumentedRoutes` was used briefly for non-test code in 2017, then removed [here](https://github.com/broadinstitute/rawls/pull/860/files#diff-7dbb8ca023296657cb3f8bdc7e7fcd59737d95cff157d294f177817a476490d5L39). @dvoet, any concerns about putting this back in?